tests/zkevm/test_worst_compute.py

```diff
@@ -170,3 +176,86 @@ def test_worst_modexp(
         post={},
         blocks=[Block(txs=[tx])],
     )
+
+
+@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize(
+    "gas_limit",
+    [
+        Environment().gas_limit,
```

Comment:
Not sure why this is needed as we later pass it to the same object?

Reply:
I think the original intention was to have other `gas_limit` parameters be relative to the `Environment().gas_limit`. So the test setup being something like: `curr_gas_limit`, `curr_gas_limit*1.5`, `curr_gas_limit*2`, ...

But I'm not sure we're going going to do that soon, so we can also simplify it for now and use the default `Environment()` below.

Reply:
Yup, this is unacceptable. Will address this in another PR that disables this warning for all tests in `./tests/zkevm`.

Reply:
But will this work without having the "gas_limit" param, by simply using the default `Environment()` instance?

Reply:
https://github.com/ethereum/execution-spec-tests/pull/1598

Reply:
I removed this param because not needed. But the warning is still there.

Reply:
This is to be expected as you set the tx's `gas_limit` to that of the `Environment` in L329:
https://github.com/ethereum/execution-spec-tests/blob/b2df5ca3c5e7a5182217d2e7a8b2cec829f3cad2/tests/zkevm/test_worst_compute.py#L326-L331

The warning comes from here:
https://github.com/ethereum/execution-spec-tests/blob/46fce2ef7e5e99065f6e61857c8b807f9c80fccc/src/ethereum_test_specs/blockchain.py#L420-L424

This will be suppressed for all tests marked with `zkevm` as #1598 currently stands.

Reply:
Merged:
- https://github.com/ethereum/execution-spec-tests/pull/1598